site/app/entities/chat/Message.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+declare(strict_types=1);
+
+namespace app\entities\chat;
+
+use DateTime;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity]
+#[ORM\Table(name: "chatroom_messages")]
+class Message {
+
+    #[ORM\Id]
+    #[ORM\Column(name: "id", type: Types::INTEGER)]
+    #[ORM\GeneratedValue]
+    private int $id;
+
+    #[ORM\ManyToOne(targetEntity: "Chatroom")]
```

Comment:
For target entity you can use the ::class syntax to specifically refer to that class. It is better for linting purposes and would be helpful if we ever changed the directory structure.